source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -545,8 +569,26 @@ void Filter::setEncoderFilterCallbacks(Http::StreamEncoderFilterCallbacks& callb
   watermark_callbacks_.setEncoderFilterCallbacks(&callbacks);
 }
 
-void Filter::sendRequest(ProcessingRequest&& req, bool end_stream) {
+void Filter::sendRequest(const ProcessorState& state, ProcessingRequest&& req, bool end_stream) {
   // Calling the client send function to send the request.
+  if (!processing_request_modifier_initialized_) {
+    processing_request_modifier_initialized_ = true;
+    processing_request_modifier_ = createProcessingRequestModifier(
```

Comment:
Is there a motivation for the lazy initialization?
Would it be better to validate and (possibly) initialize the object when the ext-proc filter is created? If the validation fails, it could then reject the ext-proc config rather than continuing to work with a broken config.